language/examples/bqml/bqml_llm_examples.ipynb

```diff
@@ -0,0 +1,944 @@
+{
```

Comment:
<p>Could you add in a blurb here on what the user should do explicitly in the cell below and what they mean?</p><p> </p><p>For example:</p><p> </p><p>Fill in the following variable names below:</p><ul><li>**DATASET_ID**: an ID of an existing BigQuery dataset</li><li>**CONN_NAME**: the name of a BigQuery connector that you will create later in this notebook</li><li>**CONN_SERVICE_ACCOUNT**: the email address of the service account that you will use for your BigQuery connection</li><li>**LLM_MODEL_NAME**:__&lt;description&gt;___</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>thanks - done</p>